### ISSUE TITLE
Request: Option to restore sidebar gesture

### ISSUE THREAD

**Issue by jxxe** [notaapp/talk#355]
It seems like the gesture that toggled the files and sidebar views (equivalent to `control + 1...3`) now navigates through history. I didn't realize how much I used on it until it was removed. I would love an option to revert back to the old functionality.

**Comment by erusev** [comment]
Yes, I miss it too. I think we can make cmd-swipe control the sidebar. Would this help?

In the meantime, you can also use cmd-\ which is a new shortcut that toggles the file manager.

**Comment by lolbat** [comment]
I haven't used Nota since this was removed.

I can't imagine why it was but it was literally keeping me from using another tool and now its gone

**Comment by lolbat** [comment]
Also on the Mac Cntl 1 etc change the workspace you are in so if you use those keyboard commands then you have to manually select the commands from a menu

**Comment by qazzym** [comment]
I agree. I loved this behaviour and was sad to see it removed. There is something about the physical gesture on the trackpad translating to a manipulation of the digital Nota canvas that was extremely satisfying.

**Comment by erusev** [comment]
We just released [0.40](https://nota.md/changelog.html) with support for ⌘+swipe. Can you see if it helps? We also loved this behavior, but we feel that people do `go back` / `go forward` more often than `toggle sidebar`, and this is why we decided to give it the better (easier) shortcut. It's also how browsers work. I hope this makes sense.

**Comment by jxxe** [comment]
Unfortunately it doesn't seem to be working for me in 0.40.

Personally, I don't think of my notes as a series of pages, horizontally aligned the way I imagine my browser history and would prefer that swipes toggled the sidebar. A settings option would be nice, but either way it's not a dealbreaker for me.

BTW: Thanks for bringing back the old tab style! It looks much cleaner. The settings tabs are a little funky though.
<img width="396" alt="SCR-20230221-lvr" src="https://user-images.githubusercontent.com/64276359/220466180-0e61a935-e23f-4553-af7b-f81a5d671e5b.png">

**Comment by lolbat** [comment]
It feels pretty clunky compared to the old style

**Comment by qazzym** [comment]
Agreed, it definitely feels clunkier and less fluid. A browser and a notes app are 2 separate things in my mind, with different conceptual models and, correspondingly, interaction patterns. 

I did also notice the settings tabs looking a bit funny too, as if their tops have been cut off prematurely.

**Comment by erusev** [comment]
Thanks for everyone's input on this, I appreciate it.

> The settings tabs are a little funky though.

I agree this can be better, I'll see if we can fix this.

**Comment by bellebethcooper** [comment]
For what it's worth, I was really glad the old swipe behaviour was removed. I have custom swipes set up on my magic mouse using Better Touch Tool that open/close tabs and move between tabs, and this swipe gesture used to interfere with those and I couldn't figure out how to turn it off.

**Comment by petrbouchal** [comment]
From what I see at https://github.com/notaapp/talk/issues/370, this is now Cmd+Swipe with pointer over left side of window.
